Apps/Medication/src/Controllers/MedicationController.cs

```diff
@@ -90,6 +91,7 @@ public RequestResult<MedicationResult> GetMedication(string drugIdentifier)
 
             RequestResult<Dictionary<string, MedicationResult>> result = new RequestResult<Dictionary<string, MedicationResult>>()
             {
+                ResultStatus = Common.Constants.ResultType.Sucess,
```

Comment:
typo Sucess > Success

Reply:
good catch. Fixed it, it propagated everywhere